plugins/DBSize/spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative './plugin.rb'
+describe 'DBSize plugin' do
+  before(:all) do
+    @db_table = Iam.DB[:db_size_measurements]
+    db_key = 'Data Base Size in Bytes'
+  end
+
+  # Store method
+  describe '.store method' do
+    before(:all) do
+      DBSize.new.register
+      @cache = Cache.new(ENV['CACHE_DIR'])
+    end
+
+    before(:each) do
+      @cache.set('test_db', db_key: "123456.0", active: true)
```

Comment:
If you want the cache to look like this we can try to do that, but the current cache is set as
```
@cache.set(var[:"DB Name"], var[:"Data Base Size in Bytes"])
```
where `var[:"DB Name"]` is the database name and `var[:"Data Base Size in Bytes"]` is the size in bytes. It's pretty bare-bones but it's all the info we have to work with for now.